metadata-builders/src/main/java/com/linkedin/metadata/builders/search/DataJobIndexBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.linkedin.metadata.builders.search;
+
+import com.linkedin.common.Ownership;
+import com.linkedin.common.urn.DataJobUrn;
+import com.linkedin.datajob.DataJobInfo;
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.data.template.StringArray;
+import com.linkedin.metadata.search.DataJobDocument;
+import com.linkedin.metadata.snapshot.DataJobSnapshot;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class DataJobIndexBuilder extends BaseIndexBuilder<DataJobDocument> {
+  public DataJobIndexBuilder() {
+    super(Collections.singletonList(DataJobSnapshot.class), DataJobDocument.class);
+  }
+
+  @Nonnull
+  private static String buildBrowsePath(@Nonnull DataJobUrn urn) {
+    return ("/" + urn.getFlowEntity().getFlowIdEntity() + "/"  + urn.getJobIdEntity()).toLowerCase();
```

Comment:
Sorry, not entirely sure what you mean.

Reply:
I think this is not possible given the information contained in the datajob urn- but it would be nice to be able construct more legible browse paths down the line. That is likely out of scope for this PR though.

Reply:
OK, got it. I'll leave it as is then. 👍